.goreleaser.yaml

```diff
@@ -123,6 +123,12 @@ docker_manifests:
       - "flashbots/suave-geth:latest-amd64"
       - "flashbots/suave-geth:latest-arm64"
 
+nightly:
+  name_template: "{{ incpatch .Version }}-nightly"
+  tag_name: nightly
```

Comment:
**doubt:**

I am not very familiar with this part of goreleaser, but maybe it should be `tag_name: latest-nightly`.